packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -239,18 +250,25 @@ class SwitchExample extends React.Component {
   }
 }
 
-class SelectionExample extends React.Component {
-  constructor(props) {
+class SelectionExample extends React.Component<
+  {},
+  {
+    isSelected: boolean,
+    isEnabled: boolean,
+  },
+> {
+  constructor(props: {}) {
     super(props);
-    this.selectableElement = React.createRef();
+    this.selectableElement = createRef();
   }
+  selectableElement: {current: React.ElementRef<any> | null};
```

Comment:
It'd be great if we could type this ref instead of using `any`! You can look at what `myRef` is passed as a ref to to find out what kind of ref it is (ex if it was <View ref={myRef} /> the type of the ref would be `React.ElementRef<typeof View>`.